src/libs/elektra/key.c

```diff
@@ -451,6 +473,7 @@ Key * keyDup (const Key * source)
  * @param dest the key which will be written to
  * @param source the key which should be copied
  *     or NULL to clean the destination key
+ * @param flags same behaviour as keyDup()
```

Comment:
use `@copydoc` to also have the relevant information here.

Reply:
Unless I missed something in the doxygen docs, `@copydoc` copies the full doxygen comment. There are `@copybrief` and `@copydetails`, but they would still copy to much information.

Reply:
See `@copydetails doxygenFlatCopy` or `@copydetails doxygenNamespaces` in src/libs/elektra/keyset.c respectrive src/libs/elektra/keyname.c how to do it.